databases/week3/queries.sql

```diff
@@ -0,0 +1,130 @@
+USE `Meal_Sharing`;
+
+SELECT
+    `id`,
+    `price`,
+    `title`
+FROM
+    `meal`;
+
+SELECT
+    *
+FROM
+    `meal`;
+
+INSERT INTO
+    `meal` (
+        `id`,
+        `title`,
+        `description`,
+        `location`,
+        `price`,
+        `when`,
+        `max-reservations`,
+        `created-date`
+    )
+VALUES
+    (
+        1,
+        `Pasta`,
+        `Pasta with chiken and cheese`,
+        `Hvidovre torv 8`,
+        200,
+        `2023.06.03`,
+        6,
+        `2023.06.02`
+    );
+
+SELECT
+    *
+FROM
+    `meal`
+WHERE
+    `id` = 1;
+
+UPDATE
+    meal
+SET
+    max_reservations = 7,
+    price = 250
+WHERE
+    id = 1;
+
+DELETE FROM
+    meal
+WHERE
+    id = 1;
+
+SELECT
+    *
+FROM
+    reservation;
+
+INSERT INTO
+    reservation (
+        `id`,
+        `number_of_guests`,
+        `meal_id`,
+        `contact_phonenumber`,
+        `contact_email`,
+        `created_date`
+    )
+VALUES
+    (
+        1,
+        2,
+        1,
+        '71328789',
+        'ati22@gmail.com',
+        '2023.06.02'
+    );
+
+SELECT
+    *
+FROM
+    `reservation`
+WHERE
+    `id` = 1;
+
+UPDATE
+    `reservation`
+SET
+    `number-of-guests` = 10
+WHERE
+    `id` = 1;
```

Comment:
Are you sure if this mechanism works as you have different column names? I don't think it works unless some super magics. 
![image](https://github.com/Atefeh-Behroozi/hyf-homework/assets/61273043/a3c509c1-dea2-4413-b7df-952c5ae7fb16)

Reply:
Sorry about this problem. I fix it.